packages/website/versioned_docs/version-legacy/faq.md

```diff
@@ -0,0 +1,95 @@
+
+## Why do I have duplicate content in my results?
+
+This can happen when you have more than one URL pointing to the same content, for example with `./docs`, `./docs/` and `./docs/index.html`.
+
+You set the `stop_urls` to all the patterns you want to exclude. The following example will exclude all URLs ending with `/` or `index.html`
+
+```json
+{
+  "stop_urls": ["/$", "/index.html$"]
+}
+```
+
+## Why are custom changes from the Algolia dashboard ineffective?
+
+Changing your setting from the dashboard might be something you want to do for some reasons .
+
+Every successful crawl sets the DocSearch settings. These settings will be overridden at the next crawl. We **do not recommend to edit anything from the dashboard**. These changes come from the JSON configuration itself.
+
+You can use the [custom_settings parameter][8] for such purpose.
+
+## A documentation website I like does not use DocSearch. What can I do?
+
+We'd love to help!
+
+If one of your favorite tool documentation websites is missing DocSearch, we encourage you to file an issue in their repository explaining how DocSearch could help. Feel free to [send us an email][1] as well, and we'll provide all the help we can.
+
+## How many records does the DocSearch crawl create?
+
+The [`nb_hits` property][8] in your configuration keeps track of the number of records the crawl has extracted and indexed by the last DocSearch run. A crawl updates this number automatically.
+
+The DocSearch scraper follows [the recommended atomic-reindexing strategy][9]. It creates a brand new temporary index to populate the data scraped from your website. When successful, the crawl overwrites the old index defined in your configuration with the key `index_name`.
+
+## Why aren't my pages indexed?
+
+We are scraping your website according to your configuration. It might happen that some pages are missing from the search. Some possible reasons for that are:
+
+- Makes sure you are not filtering on the search by wrongly using `facetFilters`. [See here for more details][10].
+- Make sure that an other indexed page references the page missing with an hyperlink tag `<a/>`.
+- Make sure you are [providing a compliant sitemap from the configuration][11] and that it references the page.
+
+## How did we build this website?
+
+We build this website with [Docusaurus v2][12]. We were helped by a great man who inspired us a lot, Endi. We want [to pay a tribute to this exceptional human being that will be always part of the DocSearch project][13]. Rest in peace mate!
+
+## What is the timeline on the v3?
+
+We are pre-releasing the v3 on docusaurus 2. It will help us to iterate faster on it and make sure we are ready to release a vanilla version. We will provide a migration guide to help you move on this new version. If you want to have more information on this version, you can [watch the search party we made about this topic][16].
```

Comment:
```suggestion
We are pre-releasing the v3 on Docusaurus 2. It will help us to iterate faster on it and make sure we are ready to release a vanilla version. We will provide a migration guide to help you move on this new version. If you want to have more information on this version, you can [watch the search party we made about this topic][16].
```

Capitalize proper noun Docusaurus.